.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,38 @@
+name: 🐞 Bug Report
+description: Create a bug report to help us improve
+title: "bug: "
+labels: ["🐞❔ unconfirmed bug"]
+assignees:
+  - oben01
+  - manuel-rw
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: input
+    id: version
+    attributes:
+      label: Chart Version
+      description: What version of the chart are you using?
+  - type: textarea
+    id: description
+    attributes:
+      label: Bug Description
+      description: Please accurately describe the bug you encountered.
+    validations:
+      required: true
+  - type: textarea
+    id: values
+    attributes:
+      label: values.yaml
+      description: |
+        Please copy and paste your `values.yaml` file. **Make sure to remove/replace all secrets!**
+      render: yaml
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: |
+        Please copy and paste and relevant Helm/Pod logs. This will be automatically formatted into code, so no need for backticks.
```

Comment:
Maybe instruct the user how to obtain logs? Shouldn't differ to much for different clusters using kubectl, no?